pkg/workflows/wasm/host/module.go

```diff
@@ -161,10 +161,12 @@ func NewModule(modCfg *ModuleConfig, binary []byte, opts ...func(*ModuleConfig))
 	// Take the max of the default and the configured max memory mbs.
 	// We do this because Go requires a minimum of 16 megabytes to run,
 	// and local testing has shown that with less than 64 mbs, some
-	// binaries may error sporadically.
+	// binaries may error sporadically. At 64 MB max memory, we have
+	// some OOM issues with compute steps while using debugger.
 	modCfg.MaxMemoryMBs = int64(math.Max(float64(defaultMaxMemoryMBs), float64(modCfg.MaxMemoryMBs)))
 
 	cfg := wasmtime.NewConfig()
+	defer cfg.Close()
```

Comment:
I think you need to add this to the module struct: the config is used in the engine which outlives this funtion.

Reply:
Can we rename it? The Close needs to be tied to the module Close, so we need to store it on the module.

Reply:
Fixed this, as per our discussion